courses/templates/partials/metadata-tiles.html

```diff
@@ -40,10 +40,10 @@
         <span class="title">FORMAT</span>
         <span class="text">Online</span>
       </li>
-      {% if course.current_price %}
+      {% if course.first_unexpired_run.current_price %}
```

Comment:
thinking, what happened in case of a program ?

Reply:
Definitely we will have to add check for programs because price is at program level. May be in #395.

Reply:
reference #410.

Reply:
@asadiqbal08 Fix for programs as well.